Cargo.toml

```diff
@@ -5,5 +5,12 @@ authors = ["Vurich <jackefransham@gmail.com>"]
 description = "(Mostly) drop-in replacement for floats that preserves the signalling bit in NaNs under conversions and unary operations"
 license = "Unlicense"
 
+[features]
+default = ["std"]
+std = []
+
+[target.'cfg(not(feature = "std"))'.dependencies]
+libm = "0.1.2"
```

Comment:
Yes, this is only for `fract`.

The `force_eval` is copied from musl: https://git.musl-libc.org/cgit/musl/tree/src/math/trunc.c

I honestly haven't looked at performance. I want this mostly for wasmi.
That being said, `libm` is a non-public dependency, so it can be switched out any time.

Reply:
Fair enough! If it's good enough for musl, it's good enough for me